DQM/L1TMonitor/src/L1TStage2uGMT.cc

```diff
@@ -5,7 +5,9 @@ L1TStage2uGMT::L1TStage2uGMT(const edm::ParameterSet& ps)
     : ugmtMuonToken(consumes<l1t::MuonBxCollection>(ps.getParameter<edm::InputTag>("muonProducer"))),
       monitorDir(ps.getUntrackedParameter<std::string>("monitorDir")),
       emul(ps.getUntrackedParameter<bool>("emulator")),
-      verbose(ps.getUntrackedParameter<bool>("verbose"))
+      verbose(ps.getUntrackedParameter<bool>("verbose")),
+      etaScale_(0.010875),
```

Comment:
hi @thomreis - please document these magic numbers (maybe they can be taken from somewhere else?